shader_source.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+
+import os
+import requests
+
+
+@dataclass
```

Comment:
Let's just use a plain ol' class for now. The init should set the api_key, but I think the shader_id should be passed into the `_request_shader(self, shader_id)` and the public `get_shader_codes(self, shader_id)`

Reply:
yeap, agree